distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestPlatform.java

```diff
@@ -43,11 +43,21 @@
           HTTPS_PORT,
           "/admin/jolokia/exec/org.codice.ddf.platform.logging.LoggingService:service=logging-service/retrieveLogEvents");
 
+  private static final DynamicUrl REPORT_GENERATION_URL =
+      new DynamicUrl(
+          DynamicUrl.SECURE_ROOT,
+          HTTPS_PORT,
+          "/services/internal/metrics/report.xls?startDate=2018-07-10T22:00:00Z&endDate=2018-07-10T23:00:00Z&summaryInterval=minute");
```

Comment:
❓ Do these times need to be more dynamic? What happens if the range doesn't exist?

Reply:
We don't need the parameters at all, that was a fat fingered paste. Removing